src/demo-app/DemoApp.tsx

```diff
@@ -26,6 +27,9 @@ function DemoApp() {
         <Route path="/hsds">
           <HsdsApp />
         </Route>
+        <Route path="/h5core">
+          <H5CoreApp />
+        </Route>
```

Comment:
A new route. We could make it the default at some point :shrug:

Reply:
Maybe when it's ready to be exported in the package?